action.py

```diff
@@ -808,23 +807,29 @@ def import_annotations(self, reader_app):
         self.selected_mi = get_selected_book_mi(self.get_options(),
                                                 msg=self.SELECT_DESTINATION_MSG,
                                                 det_msg=self.SELECT_DESTINATION_DET_MSG)
-        if not self.selected_mi:
-            return
```

Comment:
Why this was removed? What it actually does?

Not a part of #25 (not sure if it should be).